diff --git a/Sources/CCryptoBoringSSL/crypto/mem.c b/Sources/CCryptoBoringSSL/crypto/mem.c
index b1d165e..0f66bc4 100644
--- a/Sources/CCryptoBoringSSL/crypto/mem.c
+++ b/Sources/CCryptoBoringSSL/crypto/mem.c
@@ -84,7 +84,10 @@ static void __asan_unpoison_memory_region(const void *addr, size_t size) {}
 // Windows doesn't really support weak symbols as of May 2019, and Clang on
 // Windows will emit strong symbols instead. See
 // https://bugs.llvm.org/show_bug.cgi?id=37598
-#if defined(__GNUC__) || (defined(__clang__) && !defined(_MSC_VER))
+
+// Swift Crypto disables this for compatibility with weird libraries. See
+// https://github.com/apple/swift-nio-ssl/issues/189
+#if 0
 // sdallocx is a sized |free| function. By passing the size (which we happen to
 // always know in BoringSSL), the malloc implementation can save work. We cannot
 // depend on |sdallocx| being available so we declare a wrapper that falls back
