diff --git a/Sources/CCryptoBoringSSL/crypto/evp/print.cc b/Sources/CCryptoBoringSSL/crypto/evp/print.cc
index 9f322ed1..f052f89a 100644
--- a/Sources/CCryptoBoringSSL/crypto/evp/print.cc
+++ b/Sources/CCryptoBoringSSL/crypto/evp/print.cc
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <inttypes.h>
+
 #include <CCryptoBoringSSL_evp.h>
 
 #include <CCryptoBoringSSL_bio.h>

